src/connectors/scene/CryptoExchangeSceneConnector.js

```diff
@@ -102,7 +101,11 @@ export const mapStateToProps = (state: State): CryptoExchangeSceneComponentState
     forceUpdateGuiCounter: state.cryptoExchange.forceUpdateGuiCounter,
     showWalletSelectModal: state.cryptoExchange.walletListModalVisible,
     showConfirmShiftModal: state.cryptoExchange.confirmTransactionModalVisible,
-    showNextButton
+    showNextButton,
+    // Not Sure why flow does not like the gettingTransaction type.
+    // There is no reason to fail it.
+    // $FlowFixMe
+    gettingTransaction: state.cryptoExchange.gettingTransaction
```

Comment:
You need to add `gettingTransaction` to ReduxTypes.js. That will fix the Flow error

Reply:
Fixed and pushed . Thanks